### PR TITLE
Small changes to achieve more good user experience 

### PR DESCRIPTION
**Pull request by Den-creator**
Make var searchController "public". Due to this developers can make keyboard appears when countryPicker will be shown. 
SearchBarCancelButtonClicked ( delete "applySearch = false"). 
When searchBarCancelButtonClicked - dismiss CountryPikerController.

### CODE REVIEW

File: CountryPicker/CountryPicker/CountryPickerController.swift [hunk 1]
```diff
@@ -357,7 +357,6 @@ extension CountryPickerController: UISearchBarDelegate {
     public func searchBarCancelButtonClicked(_ searchBar: UISearchBar) {
         searchBar.resignFirstResponder()
         searchBar.text = ""
-        applySearch = false
```
Comment (SURYAKANTSHARMA) [inline CountryPicker/CountryPicker/CountryPickerController.swift:360]: why this deleted

File: CountryPicker/CountryPicker/CountryPickerController.swift [hunk 2]
```diff
@@ -45,7 +45,7 @@ open class CountryPickerController: UIViewController {
     
     //MARK: View and ViewController
     var presentingVC: UIViewController?
-    var searchController = UISearchController(searchResultsController: nil)
+    public var searchController = UISearchController(searchResultsController: nil)
```
Comment (SURYAKANTSHARMA) [inline CountryPicker/CountryPicker/CountryPickerController.swift:48]: why to make it public ?

**Review by SURYAKANTSHARMA — COMMENTED** [review]
@Den-creator few questions. Thanks for PR.

**Comment by SURYAKANTSHARMA** [comment]
@Sharkesm  @hardeep-singh  guys please help testing this

**Comment by SURYAKANTSHARMA** [comment]
@Den-creator any update ? could you please add some gif file as well

**Comment by Den-creator** [comment]
Sorry, I was too tired when doing this pull request. Your library works great. Thank you

**Pull request closed without merging**